### ISSUE TITLE
Exonerate multiprocessing timeout error

### ISSUE THREAD

**Issue by mossmatters** [mossmatters/HybPiper#121]
User Ryan Brewer reports an issue running Exonerate for some genes. In the log file:

```
 File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/HybPiper/2.1.1-foss-2021b/lib/python3.9/site-packages/hybpiper/assemble.py", line 1077, in exonerate_multiprocessing
    gene_name, prot_length, run_time = future.result()
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/concurrent/futures/_base.py", line 438, in result
    return self.__get_result()
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/Python/3.9.6-GCCcore-11.2.0/lib/python3.9/concurrent/futures/_base.py", line 390, in __get_result
    raise self._exception
pebble.common.ProcessExpired: Abnormal termination

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/HybPiper/2.1.1-foss-2021b/bin/hybpiper", line 8, in <module>
    sys.exit(main())
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/HybPiper/2.1.1-foss-2021b/lib/python3.9/site-packages/hybpiper/assemble.py", line 1775, in main
    args.func(args)
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/HybPiper/2.1.1-foss-2021b/lib/python3.9/site-packages/hybpiper/assemble.py", line 1461, in assemble
    exonerate_multiprocessing(genes,
  File "/cvmfs/hpc.rug.nl/versions/2023.01/rocky8/x86_64/amd/zen3/software/HybPiper/2.1.1-foss-2021b/lib/python3.9/site-packages/hybpiper/assemble.py", line 1102, in exonerate_multiprocessing
    print(f'error.traceback is: {error.traceback}')  # traceback of the function
AttributeError: 'ProcessExpired' object has no attribute 'traceback'

```

**Comment by DavCP** [comment]
Hi, @mossmatters , Any idea why this might be happening? i'm getting this error in some of my Hybpiper instances.

**Comment by chrisjackson-pellicle** [comment]
Should be fixed in HybPiper >= v2.1.6
